rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/CaseDefinitionController.java

```diff
@@ -85,6 +90,21 @@ public CaseType dataCaseworkerIdAndJurisdictionIdCaseTypeGet(
         return caseRoleService.findByCaseTypeId(caseTypeId);
     }
 
+
+    @GetMapping(value = "/data/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/access/profile/roles",
+        produces = {"application/json"})
+    @ApiOperation(value = "Get Case Roles for a case type",
+        notes = "Returns list of case roles of a single case type.\n", response = CaseRole.class)
```

Comment:
```suggestion
        notes = "Returns list of case roles of a single case type.\n", response = RoleAssignment.class)
```

Reply:
And we may want to change the description since it's no longer returning `CaseRole`?

Reply:
DONE :)